bower.json

```diff
@@ -7,12 +7,12 @@
     "consumerfinance.gov",
     "capital-framework"
   ],
-  "authors": [
-    {
-      "name": "Consumer Financial Protection Bureau",
-      "url": "https://cfpb.github.io/"
-    }
-  ],
+  "author":
+  {
+    "name": "Consumer Financial Protection Bureau",
+    "email": "tech@cfpb.gov",
+    "url": "https://cfpb.github.io/"
+  },
```

Comment:
Can't do this, according to the [spec](https://github.com/bower/bower.json-spec#authors).